app/javascript/src/javascripts/related_tag.js

```diff
@@ -0,0 +1,142 @@
+import Uploads from './uploads';
+import Utility from './utility';
+
+let RelatedTag = {};
+
+RelatedTag.initialize_all = function() {
+  $(document).on("click.danbooru", ".related-tags-button", RelatedTag.on_click_related_tags_button);
+  $(document).on("click.danbooru", ".related-tags a.search-tag", RelatedTag.toggle_tag);
+  $(document).on("click.danbooru", "#show-related-tags-link", RelatedTag.show);
+  $(document).on("click.danbooru", "#hide-related-tags-link", RelatedTag.hide);
+  $(document).on("keyup.danbooru.relatedTags", "#upload_tag_string, #post_tag_string", RelatedTag.update_selected);
+
+  $(document).on("danbooru:update-source-data", RelatedTag.on_update_source_data);
+  $(document).on("danbooru:open-post-edit-dialog", RelatedTag.hide);
+  $(document).on("danbooru:close-post-edit-dialog", RelatedTag.show);
+
+  // Initialize the recent/favorite/translated/artist tag columns once, the first time the related tags are shown.
+  $(document).one("danbooru:show-related-tags", RelatedTag.initialize_recent_and_favorite_tags);
+  $(document).one("danbooru:show-related-tags", Uploads.fetch_data_manual);
```

Comment:
Shouldn't it be **on** instead of **one**?

```javascript
 $(document).on("danbooru:show-related-tags", RelatedTag.initialize_recent_and_favorite_tags);
 $(document).on("danbooru:show-related-tags", Uploads.fetch_data_manual);
```